src/main/osd/osd_elements.c

```diff
@@ -1498,13 +1498,18 @@ static void osdElementRtcTime(osdElementParms_t *element)
 static void osdElementRssiDbm(osdElementParms_t *element)
 {
     const int8_t antenna = getActiveAntenna();
+    const int16_t osdRssiDbm = getRssiDbm();
     static bool diversity = false;
 
+    if (osdRssiDbm < osdConfig()->rssi_dbm_alarm) {
```

Comment:
If this runs in the loop, I would load the config value in the init function, as config calls are expensive correct?

There are couple of config calls, I think we should initialize the all in a local variable in init.

Reply:
Agree, was thinking the same

Reply:
These setting can be changed on the fly outside of initialisation. Config access is considered low cost as it's held in RAM. `osdConfig()->` appears in https://github.com/betaflight/betaflight/blob/321fd468fd2f3be78681d45e3e578cd768c3f26c/src/main/osd/osd_elements.c 53 times for such comparisons, or, for example, determining which unit symbols to display.